test/schemaValidation.test.ts

```diff
@@ -203,14 +203,15 @@ suite('Validation Tests', () => {
                 const content = 'cwd: hello\nan';
                 const validator = parseSetup(content);
                 validator.then(function (result) {
+                    console.log(`RESULT: ${JSON.stringify(result)}`)
                     assert.equal(result.length, 2);
                     assert.deepEqual(
                         result[0],
-                        createExpectedError(BlockMappingEntryError, 1, 13, 1, 13)
+                        createExpectedError(BlockMappingEntryError, 1, 2, 1, 2)
```

Comment:
I feel pretty confident that the old test was wrong — the error should be on character two, the line doesn't extend to a thirteenth character.

This is in line with the logging I did on the testing suite in `0.4.1`.

Reply:
![image](https://user-images.githubusercontent.com/11686766/85236481-649ebd80-b3ec-11ea-8526-a5f0ecde4336.png)

Reply:
^output from `0.4.1`